java/system/ksp-jboss-allow-module-files.yaml

```diff
@@ -0,0 +1,89 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-jboss-allow-module-files
+  namespace: default # Change your namespace
+spec:
+  tags: ["jboss", "wildfly", "configuration", "files", "modules" ]
```

Comment:
@tamilmaran-7 Workload tag is missing

Reply:
@salman-accuknox jboss is the framwework it's the workload tag

Reply:
What is the main workload tag?

Reply:
Its Java right?